multicluster/controllers/multicluster/member/node_controller.go

```diff
@@ -116,12 +119,16 @@ func (r *NodeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 	var isValidGateway bool
 
 	if stillGatewayNode {
+		gw.ServiceCIDR, err = r.getServiceCIDR()
```

Comment:
I think we should discover ServiceCIDR in NewNodeReconciler() or maybe SetupWithManager(). @luolanzone?

Reply:
A NodeReconciler without `ServiceCIDR` is invalid, so I moved the 'getServiceCIDR` step to `SetupWithManager` func.